packages/fx-core/src/component/generator/generator.ts

```diff
@@ -48,7 +48,12 @@ export class Generator {
     safeProjectNameFromVS?: string,
     targetFramework?: string,
     placeProjectFileInSolutionDir?: boolean,
-    apiKeyAuthData?: { authName: string; openapiSpecPath: string; registrationIdEnvName: string },
+    apiKeyAuthData?: {
```

Comment:
might be better to rename this also as it contains more info not limited to API Key only.

Reply:
Updated, thanks!